server/server_test.go

```diff
@@ -1587,16 +1627,10 @@ func (cli *testServerClient) runTestIssue3680(t *testing.T) {
 
 func (cli *testServerClient) runTestIssue22646(t *testing.T) {
 	cli.runTests(t, nil, func(dbt *testkit.DBTestKit) {
-		c1 := make(chan string, 1)
-		go func() {
-			dbt.MustExec(``) // empty query.
```

Comment:
it will cause ```commands out of sync. Did you run multiple statements at once``` so it has been reconstructed.

Reply:
After your change, if there's a bug that cause `dbt.MustExec("")` blocks for a long time, it fails still(but maybe in a longer time). So the change LGTM.

Reply:
After it fixed, it only blocks for a short while.